teamscale_precommit_client/precommit_client.py

```diff
@@ -185,16 +200,19 @@ def _format_findings(self, findings, branch):
         sorted_findings = sorted(findings)
 
         if self.omit_links_to_findings:
-            return ['%s:%i:1: %s: %s' % (os.path.join(self.repository_path, finding.uniformPath), finding.startLine,
-                                         self._get_finding_severity_message(finding=finding), finding.message) for
+            return ['%s:%i:1: %s: %s' % (
+                os.path.join(self.repository_path, self._remove_path_prefix(finding.uniformPath)), finding.startLine,
```

Comment:
I'm wondering whether we want to transform the findings once somewhere above, instead of transforming the uniformPath in each of the cases. Would make it less likely that we eventually have an error due to accidentally using the unmodified uniform path. What do you think?